data_pipeline/twitter_ads_api/twitter_ads_api_pipeline.py

```diff
@@ -85,36 +84,87 @@ def get_param_dict_from_api_query_parameters(
     return result
 
 
+def get_final_end_date(
+    api_query_parameters_config: TwitterAdsApiApiQueryParametersConfig,
+    initial_start_date_value: datetime
+) -> datetime:
+    yesterday_date = get_yesterdays_date()
+    end_date_by_period = initial_start_date_value + timedelta(
```

Comment:
minor naming: `end_date_of_period`? Or `period_end_date` (I often aim to put the type at the end, although sometimes that doesn't read well)

Reply:
`period_max_end_date`